services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/allu/CableReportService.kt

```diff
@@ -33,11 +35,31 @@ class CableReportService(
     private val properties: AlluProperties,
     private val ioDispatcher: CoroutineDispatcher = Dispatchers.IO,
 ) {
+    var authToken: String? = null
+    var authExpiration: Instant? = null
 
     private val baseUrl = properties.baseUrl
     private val defaultTimeout = ofSeconds(30)
     private val attachmentUploadTimeout = ofSeconds(55)
 
+    fun getToken(): String =
+        authToken?.let {
+            if (authExpiration?.isAfter(Instant.now().plusSeconds(300)) == true) {
```

Comment:
Should this 5 mins be a constant?

Reply:
Yes. thank you. I replaced this with a constant.